tests/integration/models/reddit/test_wikipage.py

```diff
@@ -17,6 +18,14 @@ def test_content_md(self):
         with self.use_cassette():
             assert page.content_md
 
+    def test_content_md__invalid_name(self):
+        subreddit = self.reddit.subreddit("reddit.com")
+        page = WikiPage(self.reddit, subreddit, "\\A")
+
+        with self.use_cassette():
+            with pytest.raises(RedditAPIException):
```

Comment:
Can you assert here that the exception value (or string) contains "INVALID_PAGE_NAME"?